orca-clouddriver/src/main/groovy/com/netflix/spinnaker/orca/clouddriver/tasks/instance/AbstractInstancesCheckTask.groovy

```diff
@@ -101,10 +101,7 @@ abstract class AbstractInstancesCheckTask extends AbstractCloudProviderAwareTask
         }
 
         seenServerGroup[name] = true
-        Collection<String> interestingHealthProviderNames = stage.execution.appConfig.interestingHealthProviderNames as Collection
-        if (interestingHealthProviderNames == null) {
```

Comment:
After talking to @duftler, it looks like this line isn't necessary due to this change here: https://github.com/spinnaker/orca/pull/957/files#diff-f68af5115f175c0b21337307aac71d46R92